code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -23,9 +23,15 @@
 		M.add_chemical_effect(CE_BLOODRESTORE, 6 * removed)
 
 	else
-		M.adjustToxLoss(1 * removed)
-		if(istype(M,/mob/living/carbon/human))
-			var/mob/living/carbon/human/H = M
+		if(ingest_mul)
+			affect_blood(M, alien, removed * ingest_mul)
```

Comment:
```suggestion
			affect_blood(M, alien, removed)
```

Psure this is unnecessary. The `removed` value here is already the result of multiplying it through with `ingest_mul`. So you're effectively doing it twice.